pkg/controller/provider/container/ovirt/client.go

```diff
@@ -33,14 +33,34 @@ type Client struct {
 	// Raw client.
 	client *libweb.Client
 	// Secret.
-	secret *core.Secret
+	secret        *core.Secret
+	created       time.Time
+	clientTimeout time.Duration
```

Comment:
I don't see a need to store both, we can have just a single field for 'client expiration', no?